ManhTien/08_BT_Mysql/sort.php

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tìm Kiếm</title>
+    <style>
+        table , td{
+            border: 1px solid #333;
+        }
+    </style>
+</head>
+<body>
+    <table >
+        <tr>
+            <td>ID</td>
+            <td>Name</td>
+            <td>Email</td>
+            <td>Phone</td>
+            <td>Address</td>
+            <td colspan="2">Option</td>
+        </tr>
+<?php
+    include_once('connect.php');
+    if(isset($_POST['sort'])) {
+        $sort = $_POST['sort'];
+        if($sort=='ASC') 
+        {
+            $sql = "select * from user ORDER BY id ASC;";
+        } 
+        else 
+            if($sort=='DESC') {
+                $sql = "select * from user ORDER BY id DESC;";
+            }
+            else {
```

Comment:
```suggestion
            } else {
```
a thấy code có vẻ ok r, còn toàn convention thôi, e check lại 1 lượt nhé

Reply:
oke a. em hơi ẩu quá nên k để ý convention lắm